dask_distance/_utils.py

```diff
@@ -0,0 +1,31 @@
+import dask
+import dask.array
+
+import dask_distance._compat
+
+
+def _bool_cmp_mtx_cnt(u, v):
+    u = dask_distance._compat._asarray(u)
+    v = dask_distance._compat._asarray(v)
```

Comment:
Should be using a relative import here. Fixing in PR ( https://github.com/jakirkham/dask-distance/pull/5 ).